test/src/test_helpers/fixtures/editor_default_aria_label.jst.eco

```diff
@@ -0,0 +1,19 @@
+<trix-editor id="editor-without-labels"></trix-editor>
+
+<label for="editor-with-aria-label"><span>Label text</span></label>
+<trix-editor id="editor-with-aria-label" aria-label="ARIA Label text"></trix-editor>
+
+<span id="aria-labelledby-id">ARIA Labelledby</span>
+<label for="editor-with-aria-labelledby"><span>Label text</span></label>
+<trix-editor id="editor-with-aria-labelledby" aria-labelledby="aria-labelledby-id"></trix-editor>
+
+<label for="editor-with-labels"><span>Label 1</span></label>
+<label for="editor-with-labels"><span>Label 2</span></label>
+<label for="editor-with-labels"><span>Label 3</span></label>
+<label>
+  <span>Label 4</span>
+  <trix-editor id="editor-with-labels"></trix-editor>
```

Comment:
Originally, I was planning on skipping the process entirely if there was a `<label>` ancestor.

In this case, we're testing that the `Label 4` is omitted from the `aria-label` value, but I'm unsure if we'd ever encounter this scenario in the wild.

What I'd imagine is more likely:

```html
<label>
  Content
  <trix-editor ...>
</label>
```

I think we discussed these types of hierarchies before, and would want to discourage them, since a `<trix-toolbar>` would be inserted as a sibling, and that means `<button>` elements within `<label>` elements, which spells trouble.

With that in mind, would it be worth adding a guard clause for _any_ `<label>` elements as ancestors?

Reply:
> would it be worth adding a guard clause for *any* `<label>` elements as ancestors?

I think it's worth discouraging if/when we share examples, but I don't think Trix needs a guard clause. It'll be pretty apparent as soon as you try using an editor+toolbar wrapped in a `<label>`: clicking the editor essentially stops working because focus gets redirected to the first toolbar button.

Also, it *is* possible to use an ancestor `<label>` if you specify a toolbar outside of it, like:
```html
<trix-toolbar id="message-toolbar"></trix-toolbar>
<label>
  Message:
  <trix-editor toolbar="message-toolbar"></trix-editor>
</label>
```